data/database.py

```diff
@@ -2149,3 +2150,5 @@ def upstream_registry_hostname(self) -> str:
 
 is_model = lambda x: inspect.isclass(x) and issubclass(x, BaseModel) and x is not BaseModel
 all_models = [model[1] for model in inspect.getmembers(sys.modules[__name__], is_model)]
+
+configure(app_config)
```

Comment:
won't this run everytime this module is imported?

Reply:
it'll be run only once on the first import and its results will be cached in sys.modules, subsequent imports reuse the cached object

Reply:
the same thing happens today with `from app import app`, `app` is executed only once even though it is imported into many places